cli/src/main/java/io/aklivity/zillabase/cli/internal/commands/start/ZillabaseStartCommand.java

```diff
@@ -1521,6 +1521,10 @@ CreateContainerCmd createContainer(
                 "RISINGWAVE_PORT=%s".formatted(risingwave[1]),
                 "CONFIG_SERVER_HOST=%s".formatted(configServer.getHost()),
                 "CONFIG_SERVER_PORT=%d".formatted(configServer.getPort()),
+                "PYTHON_UDF_SERVER_HOST=%s".formatted("udf-server-python.zillabase.dev"),
+                "PYTHON_UDF_SERVER_PORT=%d".formatted(5000),
+                "JAVA_UDF_SERVER_HOST=%s".formatted("udf-server-java.zillabase.dev"),
+                "JAVA_UDF_SERVER_PORT=%d".formatted(5001),
```

Comment:
Are these hostnames and ports not captured as constants somewhere?
I think we are also using them to inject into zilla admin config for risingwave binding for `LINK`, right?

Reply:
>Are these hostnames and ports not captured as constants somewhere?

Yes, they are not captured as constants. 

> I think we are also using them to inject into zilla admin config for risingwave binding for LINK, right?

Yes that's right